hack/install.sh

```diff
@@ -49,11 +49,18 @@ do_install() {
 			;;
 	esac
 
-	if command_exists docker || command_exists lxc-docker; then
```

Comment:
I've removed the check for `lxc-docker` because it's been _so long_ since we actually installed binaries with that name, and having Docker installed twice is going to be the _least_ of those user's problems.